docs/docs/sbom-tools.md

```diff
@@ -0,0 +1,34 @@
+For SBOM file generation, some SBOM tools are required to be installed in your environment:
+
+#### Install CycloneDX Gomod
+[cyclonedx-gomod](https://github.com/CycloneDX/cyclonedx-gomod) creates CycloneDX Software Bill of Materials (SBOM) from Go modules
```

Comment:
[cyclonedx-gomod](https://github.com/CycloneDX/cyclonedx-gomod) creates CycloneDX Software Bill of Materials (SBOM) from Go modules.

Can we add another sentence here before the code block? Enter the following content to install CycloneDX Gomod: